Assets/Scripts/SS3D/Systems/Inventory/Items/Item.cs

```diff
@@ -78,19 +77,19 @@ public class Item : InteractionSource, IInteractionTarget
 
         public Container Container => _container;
 
-        private bool _initialised = false;
```

Comment:
to me it's clearer to keep the "= false", even though it's the default value, it also points out that it should not have any other value initialised with.